app/event_source/subscribers/fdsh/insurance_policies/posted_subscriber.rb

```diff
@@ -54,17 +57,22 @@ def process_insurance_policies_posted_event_for_h36(subscriber_logger, response,
             }
           )
 
-          if result.success?
-            message = result.success
-            subscriber_logger.info "on_h36_insurance_policies_posted acked #{message.is_a?(Hash) ? message[:event] : message}"
-          else
-            subscriber_logger.info "process_h36_insurance_policies_posted_event: failure: #{error_messages(result)}"
-          end
+          result = if enqueue.success?
+                     message = enqueue.success
+                     subscriber_logger.info "on_h36_insurance_policies_posted acked #{message.is_a?(Hash) ? message[:event] : message}"
+                     message
+                   else
+                     failure = error_messages(enqueue)
+                     subscriber_logger.info "process_h36_insurance_policies_posted_event: failure: #{failure}"
+                     failure
+                   end
           subscriber_logger.info "process_h36_insurance_policies_posted_event: ------- end"
+          result
```

Comment:
@ saipraveen18 We don't need to return the result from this method as it is not being used. So you can remove lines 70, 67, and 63 and remove the assignment in line 60.